### DISCUSSION TITLE
The title of the article is displayed above the picture in the recommended articles

### DISCUSSION THREAD

**Discussion by vlafriday** [tangly1024/NotionNext#1936] [Q&A]
Hello, I have a preview with the text in the recommended articles on the post page. But the title of the article is displayed above the picture. I wanted to ask if I can disable the display of text above the preview. So that only the picture is visible. Where is it configured?

Here https://www.vlafriday.space/my-set-for-a-25-year-old-grandfather

Screenshot 
![Screenshot_711](https://github.com/tangly1024/NotionNext/assets/78411322/b34486c6-03a8-42bb-9fd1-c8aecc0c7e5c)

**Comment by tangly1024** [comment]
https://github.com/tangly1024/NotionNext/issues/1937
